sous-chef/sous_chef/recipe_book/read_recipe_book.py

```diff
@@ -157,6 +210,33 @@ def _check_total_time(recipe: pd.Series):
         if recipe.time_total is pd.NaT:
             raise RecipeTotalTimeUndefinedError(recipe_title=recipe.title)
 
+    def _construct_filter(self, row: pd.Series, filter_str: str):
+        def _replace_entity(entity_name: str, match_obj: re.Match) -> str:
+            if (entity := match_obj.group(1)) is not None:
+                if (
+                    entity.lower()
+                    not in {
+                        "tag": self.tag_tuple,
```

Comment:
Is it possible to use getattr instead where the variable is used instead of creating a dict?

Reply:
good point, changed it to that.